Sources/PluginLibrary/Descriptor+Extensions.swift

```diff
@@ -74,6 +74,22 @@ extension FieldDescriptor: ProvidesLocationPath, ProvidesSourceCodeLocation {
     path.append(Int32(index))
   }
 
+  /// Returns true if the type can be used for a Packed field.
+  static func isPackable(type: Google_Protobuf_FieldDescriptorProto.TypeEnum) -> Bool {
```

Comment:
Style wise, I'm not sure what was best here, I went with a static on FieldDescriptor because while this is sorta an attribute of the Type enum, it is only relevant within the context of when it is a repeated field.  I can move it over to an extension if that seems better.